tests/scripts/local_monitoring_tests/authentication_tests.sh

```diff
@@ -51,97 +33,149 @@ tokens=(
   ["invalid"]="random-invalid-token-string"
   ["empty"]=""
 )
+# Tests to validate authentication types in Kruize
+function authentication_tests() {
```

Comment:
Can you share the results run

Reply:
[authentication_tests.log](https://github.com/user-attachments/files/17898199/authentication_tests.log)

Reply:
Also, I think it will be good to invoke some code that makes use of the token along with checking the pod log, you can add it as an enhancement to the test.

Reply:
Created #1386 to track this